ethereum_clients/ControlledProcess.js

```diff
@@ -1,3 +1,4 @@
+const { Notification } = require('electron')
```

Comment:
violates separation of concerns: process and notification logic should not be mixed

Reply:
This is [main process os desktop Notifications](https://electronjs.org/docs/api/notification), used to pass to the plugin so it can create notifications when it receives relevant new data.